lib/src/entrypoint.dart

```diff
@@ -742,10 +743,19 @@ class Entrypoint {
   }
 
   /// Saves a list of concrete package versions to the `pubspec.lock` file.
+  ///
+  /// Will use Windows line endings (`\r\n` if a `pubspec.lock` exists, and uses
+  /// that.
   void _saveLockFile(SolveResult result) {
-    _lockFile = result.lockFile;
-    var lockFilePath = root.path('pubspec.lock');
-    writeTextFile(lockFilePath, _lockFile.serialize(root.dir));
+    _lockFile =
+        result.lockFile(windowsLineEndings: lockFile.windowsLineEndings);
```

Comment:
did you miss something here, didn't you remove the `windowsLineEndings` state?

Reply:
Yes - I think I made a bad merge...

Fixed